docs/references/account/list-identities.md

```diff
@@ -0,0 +1 @@
+Get currently logged in user list of identities.
```

Comment:
```suggestion
Get the list of identities for the currently logged in user.
```

Reply:
I matched the convention of some of the other descriptions like list-sessions.md so I'll update that too like:

```diff
-Get currently logged in user list of active sessions across different devices.
+Get the list of active sessions across different devices for the currently logged in user.
```

and how about these two?

account/get-prefs.md:

```diff
-Get currently logged in user preferences as a key-value object.
+Get the preferences as a key-value object for the currently logged in user
```

account/get.md:

```diff
-Get currently logged in user data as JSON object.
+Get currently logged in user.
```

Reply:
Yeap lets do it 👍

Reply:
updated!